contracts/test/gateway/MockGatewayBase.sol

```diff
@@ -125,4 +131,23 @@ contract MockGatewayBase is GatewayBase {
 
         return true;
     }
+
+    /**
+     *  @notice Get the storage root for the given block height. This is only
+     *          testing
+     *
+     *  @param _blockHeight Block height for which the storage root is to be
+     *                      fetched.
+     *
+     *  @return storageRoot_ Storage root for the given block height.
+     */
+    function getStorageRoot(
```

Comment:
This should probably be moved to GatewayBase.sol

Reply:
Let us evaluate and create a ticket for this.

Reply:
OK